napari/layers/points/_points_utils.py

```diff
@@ -264,7 +300,14 @@ def coerce_symbols(array: Iterable) -> np.ndarray:
     array : np.ndarray
         Array of strings matching Symbol values.
     """
```

Comment:
This docstring should be fixed. My proposition is not best, but maybe @brisvag or @psobolewskiPhD will have a better idea? 

```suggestion
    symbol : str | Symbol | Sequence[str | Symbol],
        symbol used for rendering points. Global or per point.
    Returns
    -------
    array of Sybol
    	normalized representation of array. 
    """
```